microshift_updating/microshift-about-updates.adoc

```diff
@@ -6,43 +6,26 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-You can update a {product-title} cluster by using the OpenShift CLI (`oc`).
-// This PR is for the book build. Note that the OCP structure consists of a landing page of xrefs to other major sections within the book. MicroShift likely does not require that depth of structure, so starting simply with one page.
+Upgrades are supported on {product-title-first} beginning with the General Availability version {ocp-version}. Supported upgrades include those from one minor version to the next in sequence, for example, from 4.14 to 4.15. Patch updates are also supported from z-stream to z-stream, for example 4.14.0 to 4.14.1.
 
 [id="microshift-about-updates-understanding-microshift-updates"]
 == Understanding {product-title} updates
-{product-title} updates are available as either RPMs or by embedding the {product-title} image in an RPM OSTree blueprint.
-You can update an {product-title} cluster by using the OpenShift CLI (`oc`).
-//Platform administrators can view new update options by looking at the output of the `oc adm upgrade` command.
-//An update begins when...
-
-[NOTE]
-====
-Operators previously installed must be reinstalled using manifests.
-====
-
-[id="microshift-about-updates-rpm-updates"]
-=== RPM updates
-Using the RPM update method replaces your existing version. No rollback is possible with this update type.
-//we can call a module here or xref out; not sure the best method for our use case until we have the content
+{product-title} updates can be done by embedding the {product-title} image in an RPM OSTree blueprint, or by manually updating the RPMs. Only RPM OSTree upgrades include automatic rollbacks with Greenboot health checks.
```

Comment:
@pmtk I agree, I have this question also. WDYT @dhellmann and @DanielFroehlich Can we just use RHEL for Edge instead of `rpm-ostree` everywhere? We can define it like the RHEL docs do at the beginning: ["A RHEL for Edge image is an rpm-ostree image that includes system packages to remotely install RHEL on Edge servers."](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/introducing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images)

Reply:
Using "RHEL for Edge" works for me.